File: src/main/java/vini2003/xyz/bambootweaks/registry/BambooTweaksItemGroups.java
package vini2003.xyz.bambootweaks.registry;

import net.fabricmc.fabric.api.client.itemgroup.FabricItemGroupBuilder;
import net.minecraft.item.ItemGroup;
import net.minecraft.item.ItemStack;
import vini2003.xyz.bambootweaks.BambooTweaks;

public class BambooTweaksItemGroups {
	public static final ItemGroup BAMBOO_TWEAKS = FabricItemGroupBuilder.create(BambooTweaks.identifier("bambootweaks")).icon(() -> new ItemStack(BambooTweaksBlocks.BAMBOO_BLOCK)).build();
	
	public static void initialize() {
	
	}
}

File: common/src/main/kotlin/vini2003/xyz/bambootweaks/common/registry/BambooTweaksItemGroups.kt
package vini2003.xyz.bambootweaks.common.registry

import me.shedaniel.architectury.registry.CreativeTabs
import net.minecraft.item.ItemGroup
import net.minecraft.item.ItemStack
import vini2003.xyz.bambootweaks.CommonBambooTweaks

object BambooTweaksItemGroups {
	fun init() {
	
	}
	
	val BambooTweaks: ItemGroup = CreativeTabs.create(CommonBambooTweaks.identifier(CommonBambooTweaks.Id)) { ItemStack(
		BambooTweaksBlocks.Bamboo.orNull!!
	) }
}